source/administration/command-line-tools.rst

```diff
@@ -655,3 +660,181 @@ platform version
     .. code-block:: none
 
       platform version
+
+Mattermost 3.5 and earlier
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Typing ``platform -help`` brings up documentation for the CLI tool.
+
```

Comment:
This doesn't work if you simply cd into the bin directory and type ``platform -help``.  You need to use ``./platform -help``. Also, you need to be a user that has permissions to do this, so the command should be ``sudo ./platform -help``. Similarly for all commands, both for 3.6+ and for 3.5 and earlier.

Experienced Linux admins will know this already. But experienced Windows admins don't know the ins and outs of the Linux command line, and when you tell them to do ``platform -help`` they'll wonder for hours why they get the "platform command can't be found error". And when they do finally figure it out, they'll wonder why the thing appears to crash when you invoke any command other than -help without sudo.